utils/config.go

```diff
@@ -640,6 +640,7 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["PasswordRequireNumber"] = strconv.FormatBool(*c.PasswordSettings.Number)
 	props["PasswordRequireSymbol"] = strconv.FormatBool(*c.PasswordSettings.Symbol)
 	props["CustomUrlSchemes"] = strings.Join(*c.DisplaySettings.CustomUrlSchemes, ",")
+	props["LTIAuthEnabled"] = strconv.FormatBool(c.LTISettings.Enable)
```

Comment:
Check if we need to add this in `GenerateLimitedClientConfig` instead as the value is required when the user is not already logged in.

Reply:
Done. Have verified that the flag is received in an unauthenticated API call.